.circleci/fail_fast.sh

```diff
@@ -22,11 +22,24 @@ set -e
 RED() { echo -e "\n\033[0;31m$1\033[0m"; }
```

Comment:
Once this works, update the top-level comment on line 17 to mention the emailing. And maybe add  a reference link for the sendgrid format too?

Reply:
Done.